FortnitePorting/Plugins/Blender/FortnitePorting/import_task.py

```diff
@@ -705,18 +705,6 @@ def get_meta(search_props):
                 bpy.ops.object.mode_set(mode='OBJECT')
                 armature_modifier: bpy.types.ArmatureModifier = first(imported_mesh.modifiers, lambda mod: mod.type == "ARMATURE")
 
-                # Grab reference pose data
-                reference_pose = meta.get("ReferencePose", dict())
```

Comment:
Since this is unused data AFAIK, I'm curious if it should also be excluded from output in the `meta` object?

Reply:
yeah I mean if it's unused we can def remove it

half of the time that info is empty anyways sooooo